test/Microsoft.AspNetCore.Mvc.Test/MvcServiceCollectionExtensionsTest.cs

```diff
@@ -31,6 +31,8 @@
 using Microsoft.Extensions.Options;
 using Moq;
 using Xunit;
+using Newtonsoft.Json.Serialization;
+using Newtonsoft.Json;
```

Comment:
nit: Sort the using statements. System namespaces first then other namespaces in alphabetical order.